### ISSUE TITLE
Errors in source strings

### ISSUE THREAD

**Issue by milotype** [olivierkes/manuskript#1079]
Please correct the source strings at Hosted Weblate. I suspect it should be an ellipsis character. Probably some kind of encoding error …

1. https://hosted.weblate.org/translate/manuskript/translations/hr/?q=%C3%A2%C2%80%C2%A6&sort_by=-priority%2Cposition&offset=1
2. https://hosted.weblate.org/translate/manuskript/translations/hr/?q=%C3%A2%C2%80%C2%A6&sort_by=-priority%2Cposition&offset=2
3. https://hosted.weblate.org/translate/manuskript/translations/hr/?q=%C3%A2%C2%80%C2%A6&sort_by=-priority%2Cposition&offset=3
4. https://hosted.weblate.org/translate/manuskript/translations/hr/?q=%C3%A2%C2%80%C2%A6&sort_by=-priority%2Cposition&offset=4

**Comment by TheJackiMonster** [comment]
What really confuses me is that this issue only seems to affect some of the translation files... 

Rebuilding the files with the usual `make i18n` does not seem to affect all .ts files either. So I will probably need to change the whole process to build/repair those files since multiple languages do not update at all. I will probably need some time learning how to use .qm and .ts files properly from the start and adjust everything afterwards. Hacking in a fix might affect other translations in a bad way.

**Comment by milotype** [comment]
As mentioned, I suspect it's an encodning issue, independatly of language.
Is it meant as an ellipsis character? If so, use the Unicode character `…`

**Comment by TheJackiMonster** [comment]
> As mentioned, I suspect it's an encodning issue, independatly of language. Is it meant as an ellipsis character? If so, use the Unicode character `…`

Yes, that's the issue. However those files are partially auto-generated. So fixing it now at some occurrences won't help in the future. Instead the actual reason needs to be tracked down what's causing it.

**Comment by TheJackiMonster** [comment]
Since replacing the character might break existing translations, I'll postpone this issue to a later release.
